pkg/operator2/route.go

```diff
@@ -127,31 +126,6 @@ func routerSecretToSNI(routerSecret *corev1.Secret) []configv1.NamedCertificate
 	return out
 }
 
-func routerSecretToCA(route *routev1.Route, routerSecret *corev1.Secret, ingress *configv1.Ingress) []byte {
```

Comment:
Deleting this is wrong.  If you do not have a CA bundle in the optional CA, you still need this to trust the route.

Reply:
Would it make more sense not to make it optional, given it should always exist?

Reply:
Even if it exists, I do not think we can guarantee that it will work against the route, whereas the the current code will work as long as there is no TLS intercepting proxy.  Merging the optional CA bundle with output of `routerSecretToCA` is probably the correct approach.